_includes/head.html

```diff
@@ -1,39 +1,58 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <meta name="description" content="Connecting open source & design! 🎨" />
+  <meta name="keywords" content="connecting,open,source,and,design,osd" />
+  <meta name="author" content="" />
 
-    <title>{% if page.title %}{{page.title}} - {% endif %}Open Source Design</title>
-    <link href="/css/lora.css" rel="stylesheet" type="text/css" />
-    <link href="/css/montserrat.css" rel="stylesheet" type="text/css" />
+  <title>
+    {% if page.title %}{{ page.title }} - {% endif %}Open Source Design
+  </title>
+  <link href="/css/lora.css" rel="stylesheet" type="text/css" />
+  <link href="/css/montserrat.css" rel="stylesheet" type="text/css" />
 
-    <link rel="shortcut icon" href="/images/favicon.ico" />
-    <link rel="apple-touch-icon-precomposed" href="/images/opensourcedesign-logo.png" />
-    <link rel="mask-icon" sizes="any" href="/images/opensourcedesign-logo.svg" color="#000" />
+  <link rel="shortcut icon" href="/images/favicon.ico" />
+  <link
+    rel="apple-touch-icon-precomposed"
+    href="/images/opensourcedesign-logo.png"
+  />
+  <link
+    rel="mask-icon"
+    sizes="any"
+    href="/images/opensourcedesign-logo.svg"
+    color="#000"
+  />
 
-    <link href="/css/main.css" rel="stylesheet" />
-    
-    <!-- Facebook metadata -->
-    <meta property="og:url" content="https://opensourcedesign.net" />
-    <meta property="og:type" content="website" />
-    <meta property="og:title" content="Open Source Design" />
-    <meta property="og:image:secure_url" content="https://opensourcedesign.net/images/opensourcedesign-logo.png" />
-    <meta property="og:description" content="Connecting open source & design! 🎨" />
-    <meta property="og:site_name" content="Open Source Design" />
-    <meta property="og:locale" content="en_US" />
-    
-    <!-- Twitter metadata -->
-    <meta name="twitter:card" content="summary" />
-    <meta name="twitter:site" content="@opensrcdesign" />
-    <meta name="twitter:description" content="Connecting open source & design! 🎨" />
-    
-    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
-    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
-    <!--[if lt IE 9]>
-        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
-        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
-    <![endif]-->
+  <link href="/css/main.css" rel="stylesheet" />
+
+  <!-- Facebook metadata -->
+  <meta property="og:url" content="https://opensourcedesign.net" />
+  <meta property="og:type" content="website" />
+  <meta property="og:title" content="Open Source Design" />
+  <meta
+    property="og:image:secure_url"
+    content="https://opensourcedesign.net/images/opensourcedesign-logo.png"
+  />
+  <meta
+    property="og:description"
+    content="Connecting open source & design! 🎨"
+  />
```

Comment:
Same here, this is usually not on separate lines, as the other ones on top are neither. Seems to be an indent issue?